compliance/collection/nodeinventorizer/nodeinventory.go

```diff
@@ -1,157 +1,10 @@
 package nodeinventorizer
 
 import (
-	"time"
-
-	timestamp "github.com/gogo/protobuf/types"
-	"github.com/stackrox/rox/compliance/collection/metrics"
 	"github.com/stackrox/rox/generated/storage"
-	"github.com/stackrox/rox/pkg/uuid"
-	"github.com/stackrox/scanner/database"
-	scannerV1 "github.com/stackrox/scanner/generated/scanner/api/v1"
-	"github.com/stackrox/scanner/pkg/analyzer/nodes"
-	"golang.org/x/exp/maps"
 )
 
 // NodeInventorizer is the interface that defines the interface a scanner must implement
 type NodeInventorizer interface {
 	Scan(nodeName string) (*storage.NodeInventory, error)
 }
```

Comment:
This file can be deleted and this interface def could be moved to `main.go` and made unexported (maybe).

Reply:
I was able to unexport it, done.